src/dlgConnectionProfiles.cpp

```diff
@@ -1547,17 +1547,18 @@ void dlgConnectionProfiles::loadProfile(bool alsoConnect)
     if (entries.isEmpty()) {
         firstTimeLoad = true;
     } else {
-        QString fileName{qsl("%1%2").arg(folder, profile_history->itemData(profile_history->currentIndex()).toString())};
-        QFile file(fileName);
+        QFile file(qsl("%1%2").arg(folder, profile_history->itemData(profile_history->currentIndex()).toString()));
         file.open(QFile::ReadOnly | QFile::Text);
         XMLimport importer(pHost);
 
         qDebug() << "[LOADING PROFILE]:" << file.fileName();
         if (auto [success, message] = importer.importPackage(&file, nullptr); !success) {
             pHost->postMessage(tr("[ ERROR ] - Something went wrong loading your Mudlet profile and it could not be loaded.\n"
-                "Try loading an older version in 'Connect - Options - Profile history' or double-check that %1 looks correct.", "%1 is the filename").arg(fileName));
+                "Try loading an older version in 'Connect - Options - Profile history' or double-check that %1 looks correct.").arg(file.fileName()));
```

Comment:
You haven't moved this  disambiguation string to a preceding Qt specific comment (as per #5522!) e.g. `//: %1 is the path and file name (i.e. the location) of the problem file...

Reply:
Applied.

Reply:
Really? I'm not seeing it yet...!